maven-plugin/src/main/java/org/sonatype/nexus/maven/staging/ArtifactInfo.java

```diff
@@ -64,11 +70,11 @@ public void setTag(final String tag) {
 
 
   public String getClassifier() {
-    return classifier;
+    return NOT_APPLICABLE.equals(classifier) ? null: classifier;
   }
 
   public void setClassifier(final String classifier) {
-    this.classifier = classifier;
+    this.classifier = classifier != null ? classifier : NOT_APPLICABLE;
```

Comment:
Would it be simpler to just convert "n/a" to null and store the value internally as null. Then only the setters would need special logic and not the getters.

Reply:
Possibly - but I wasn't sure about the (de)serialisation magic and didn't want to break that, figuring that perhaps having the NOT_APPLICABLE value being serialised had some value over 'null'.

Reply:
Changed to null in [ead71ff](https://github.com/sonatype/nxrm3-maven-plugin/pull/36/commits/ead71ffee2f92e843ac341fe6d8de9bf3cbce424)